testing/skia_gold_client/lib/skia_gold_client.dart

```diff
@@ -419,15 +447,26 @@ class SkiaGoldClient {
     return imageBytes;
   }
 
+  String _getEngineCheckoutPath() {
```

Comment:
Can you compute this using `this.workDirectory` and the `Engine.findWithin` constructor in https://github.com/flutter/engine/blob/main/tools/pkg/engine_repo_tools/lib/engine_repo_tools.dart#L87.

Reply:
TIL, yes this should work nicely. Trying it.